src/Symfony/Bundle/FrameworkBundle/Controller/ControllerNameParser.php

```diff
@@ -60,19 +67,16 @@ public function parse($controller)
                 $originalController
             );
 
-            if ($alternative = $this->findAlternative($bundleName)) {
+            $provider = new AlternativeBundleNameProvider($this->kernel);
+
+            if ($alternative = $provider->findAlternative($bundleName)) {
                 $message .= sprintf(' Did you mean "%s:%s:%s"?', $alternative, $controller, $action);
```

Comment:
Is there only 1 alternative? I can imagine there could be multiple alternatives. Assuming 1 alternative is the status quo

Reply:
I've just moved this responsibility (https://github.com/symfony/symfony/blob/master/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerNameParser.php#L107-L132) out to the separate class since it could be reused in many bundle guessing snippets

It's logic is about finding **most** similar bundle name, so this definetely only 1. We can return prioritized list here leaving formatting to the end-user code